HoneyBadgers.Logic/Data.cs

```diff
@@ -0,0 +1,134 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Serialization;
+
+namespace HoneyBadgers.Logic
+{
+    public class Data
+    {
+        public List<User> Users = new();
+        public List<Movie> Movies = new();
+
+        public Data()
+        {
+            
+        }
+
+        public void LoadData(string userFilePath, string movieFilePath)
+        {
+            LoadUsers(userFilePath);
+            LoadMovies(movieFilePath);
+            InitMockData();
+        }
+        public void LoadUsers(string fileName)
+        {
+            var path = Directory.GetCurrentDirectory() + "/Resources/" + fileName;
+            using var file = new StreamReader(path);
+            try
+            {
+                var json = file.ReadToEnd();
+                var serializerSettings = new JsonSerializerSettings
+                {
+                    ContractResolver = new CamelCasePropertyNamesContractResolver()
+                };
+                var users = JsonConvert.DeserializeObject<List<User>>(json, serializerSettings);
+                if (users != null && users.Count > 0)
+                {
+                    Users = users;
+                }
+            }
+            catch (Exception)
+            {
+                Console.WriteLine("Problem reading users.json file");
+            }
+        }
+
+        public void LoadMovies(string fileName)
+        {
+            var path = Directory.GetCurrentDirectory() + "/Resources/" + fileName;
+            using var file = new StreamReader(path);
+            try
+            {
+                var json = file.ReadToEnd();
+                var serializerSettings = new JsonSerializerSettings
+                {
+                    ContractResolver = new CamelCasePropertyNamesContractResolver()
+                };
+                var moviesDto = JsonConvert.DeserializeObject<List<MovieDto>>(json, serializerSettings);
+                if (moviesDto != null && moviesDto.Count > 0)
+                {
+                    Movies = moviesDto.Select(MapMovie).ToList();
+                }
+            }
+            catch (Exception)
+            {
+                Console.WriteLine("Problem reading movies.json file");
+            }
+        }
+
+        private Movie MapMovie(MovieDto movieDto)
+        {
+            return new Movie
+            {
+                Title = movieDto.Title,
+                Year = movieDto.Year,
+                Director = movieDto.Director,
+                Writer = movieDto.Writer,
+                Actors = movieDto.Actors.Split(",").ToList(),
+                Plot = movieDto.Plot,
+                Genre = movieDto.Genre.Split(",").ToList(),
```

Comment:
In Actors and Genre Split() method remain whitespace (" ") at beginning second, third etc elements. 

`.Split(",").Select(p => p.Trim()).ToList()`
That Trim() each element 😄